QAction_1000/IfTable.cs

```diff
@@ -281,12 +281,12 @@ public IfTableGetter(SLProtocol protocol)
 
 			public object[] RateData { get; private set; }
 
-			public bool IsSnmpAgentRestarted { get; private set; }
```

Comment:
I actually don't see the need for this change. While I see the point of changing the discreets within protocol.xml so that value speaks for itself rather than true/false, I don't see the need of changing any of the C# part. Since we know value 0 means no restart and value 1 means restart, I would keep converting the param value to the IsSnmpAgentRestarted boolean variable. Let's discuss it when you are back.

BTW, if changing the var to enum like you did (again, let's first agree if we do want to do that or not), then we should also rename the var accordingly from "IsSnmpAgentRestarted" to "SnmpAgentRestartFlag" or so.